org.opent2t.sample.binaryswitch.superpopular/com.wink.binaryswitch/js/tests/test.js

```diff
@@ -1,13 +1,10 @@
 var test = require('ava');
 var OpenT2T = require('opent2t').OpenT2T;
-var config = require('./testConfig');
-
-console.log("Config:");
-console.log(JSON.stringify(config, null, 2));
-
 var translatorPath = require('path').join(__dirname, '..');
+var runBinarySwitchTests = require('opent2t-device-binaryswitch/binarySwitchTests');
+var config = require('./testConfig');
 var hubPath = require('path').join(__dirname, '../../../../org.opent2t.sample.hub.superpopular/com.wink.hub/js');
-var translator = undefined;
+var deviceId = "F85B0738-6EC0-4A8B-A95A-503B6F2CA0D8";

```

Comment:
Two reasons for this.
1.  That ID is actually hard-coded in the thingTranslator.
2.  It's the same ID that was already being passed in (look at the old implementation of GetPower and SetPower tests).  I just pulled it out into a var.

---
In reply to: [90578036](https://github.com/openT2T/translators/pull/135#discussion_r90578036) [](ancestors = 90578036)